app/src/main/java/ar/com/wolox/android/example/model/News.kt

```diff
@@ -1,10 +1,26 @@
 package ar.com.wolox.android.example.model
 
+import com.google.gson.annotations.SerializedName
+import org.joda.time.format.DateTimeFormat
+import org.ocpsoft.prettytime.PrettyTime
+import java.util.Locale
+
 data class News(
     var id: Int,
+    @SerializedName("userId")
     var userId: Int,
+    @SerializedName("createdAt")
     var createdAt: String,
     var title: String,
     var picture: String,
     var text: String
-)
\ No newline at end of file
+) {
+
+    fun getTimeFormated(): String {
+        val formatter = DateTimeFormat.forPattern("yyyy-MM-dd'T'HH:mm:ss.SSSZ")
+        val dateTime = formatter.parseDateTime(createdAt)
+        val prettyTime = PrettyTime(Locale.getDefault())
+        val ago: String = prettyTime.format(dateTime.toDate())
```

Comment:
It's not really necessary to declare `ago` as a `String`, Kotlin will interpreted alone